sigmastate/src/main/scala/sigmastate/interpreter/Interpreter.scala

```diff
@@ -1,46 +1,94 @@
 package sigmastate.interpreter
 
-import java.lang.{Math => JMath}
 import java.util
```

Comment:
unused import

Reply:
it is actually used, IntelliJ error. Try to remove and compile